### PR TITLE
Add GHA example

### PR DESCRIPTION
**Pull request by henryiii**
First point of #301.

### CODE REVIEW

File: docs/options.md [hunk 1]
```diff
@@ -42,6 +42,13 @@ variables:
   CIBW_TEST_COMMAND: "nosetests {project}/tests"
 ```
 
+> .github/workflows/*.yml ([docs](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables)) (can be global, in job, or in step)
+```yaml
+env:
+  CIBW_TEST_REQUIRES: nose
+  CIBW_TEST_COMMAND: "nosetests {project}/tests"
```
Comment (henryiii) [inline docs/options.md]:
```suggestion
  CIBW_TEST_COMMAND: nosetests {project}/tests
```

Quotes are technically not needed here.

Comment (henryiii) [reply docs/options.md]: Also, shouldn't we change the test command example to pytest?

Comment (joerick) [reply docs/options.md]: Quotes aren't needed, but I think they're useful to people unfamiliar to YAML. But yes, pytest would probably be more familiar to most readers. If you change, could you change all the CI platform examples?

**Comment by henryiii** [comment]
Travis failure seems to be unrelated, this only changes docs.

**Comment by YannickJadoul** [comment]
Restarted the test, and it's indeed green now! :-)

**Pull request merged**